osu.Game/Screens/Play/GameplayClockContainer.cs

```diff
@@ -33,6 +33,11 @@ public class GameplayClockContainer : Container
 
         public readonly BindableBool IsPaused = new BindableBool();
 
+        public override bool PropagateNonPositionalInputSubTree => !BlockGameplayInput;
+        public override bool PropagatePositionalInputSubTree => !BlockGameplayInput;
+
+        public bool BlockGameplayInput { get; set; }
```

Comment:
xmldoc. also there's no way this should be in `GameplayClockContainer` (a container managing a clock, not input).

Reply:
I see...